recipes/firebase-admin/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "firebase-admin" %}
+{% set version = "5.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 05e0c12e37a613b6aeeb2a96d4259badf139f977cb2ac630fd364722ee02a74d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - cachecontrol >=0.12.6
+    - google-api-core[grpc] >=1.22.1,
```

`[…]` doesn't work in conda as in pip. 
```suggestion
    - google-api-core >=1.22.1
```